### ISSUE TITLE
Wrong browser name (Opera) detected for some Android Oreo builds

### ISSUE THREAD

**Issue by arshadashrafi** [lancedikson/bowser#221]
`window.navigator.userAgent` of the browser is: 
~~~
Mozilla/5.0 (Linux; Android 8.0.0; LG-H870 Build/OPR1.170623.032) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.68 Mobile Safari/537.36 
~~~
 And it's detected like a 
~~~
{  
   "name":"Opera",
   "opera":true,
   "version":"",
   "webkit":true,
   "android":true,
   "osversion":"8.0.0",
   "mobile":true,
   "c":true
}
~~~
 But real name of the browser is `Chrome`

**Comment by arshadashrafi** [comment]
Note the `OPR1.170623.032` part in the user agent string. Which matches with this line https://github.com/lancedikson/bowser/blob/master/src/bowser.js#L59
and therefore detected as Opera.

I assume this bug will occur on all Chrome browsers (or maybe other browsers as well) having Android builds starting with `OPR`. See https://source.android.com/setup/start/build-numbers#source-code-tags-and-builds

**Comment by lancedikson** [comment]
@arshadashrafi, hi, it seems you have an older version of bowser library. We have fixed this issue since #199 issue, and I've double checked the UserAgent string you've provided above with the latest version of bowser and it detects it as Chrome:

```
{ name: 'Chrome',
  chrome: true,
  version: '67.0',
  blink: true,
  android: true,
  osname: 'Android',
  osversion: '8.0.0',
  mobile: true,
  a: true }
```

**Comment by arshadashrafi** [comment]
@lancedikson you are right. thanks! I was testing with an old minified version of bower.
